relay-dynamic-config/src/global.rs

```diff
@@ -1,9 +1,55 @@
+use relay_general::store::MeasurementsConfig;
 use serde::{Deserialize, Serialize};
 
 /// A dynamic configuration for all Relays passed down from Sentry.
 ///
 /// Values shared across all projects may also be included here, to keep
 /// [`ProjectConfig`](crate::ProjectConfig)s small.
-#[derive(Default, Clone, Debug, Serialize, Deserialize)]
+#[derive(Default, Clone, Debug, Serialize, Deserialize, PartialEq)]
 #[serde(default, rename_all = "camelCase")]
-pub struct GlobalConfig {}
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct GlobalConfig {
+    /// Configuration for measurements normalization.
+    pub measurements: MeasurementsConfig,
```

Comment:
Let's make all fields optional for future compatibility (e.g. we decide to discontinue this key in the future).